codice/alliance
# CAL-308 Updated NSILI mapping of subjectCategoryCode

#### What does this PR do?
Changes the NSIL Transformer to map COMMON.subjectCategoryCode to the isr.nato-reporting-code
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@dcruver @troymohl 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith
#### How should this be tested?
Verify that the value(s) from subjectCategoryCode are set in isr.nato-reporting-code
(don't have test case available, verify the unit test and expected results and that it passes)
#### Any background context you want to provide?
#### What are the relevant tickets?
CAL-308 (https://codice.atlassian.net/browse/CAL-308)
#### Screenshots (if appropriate)
#### Checklist:
